packages/blueprints/blueprint/src/ui-selectors/dynamic-kv-input.ts

```diff
@@ -20,12 +20,12 @@
 
 */
 export type DISPLAY_COMPONENT_TYPE =
-    'string' | 'number' | 'checkbox' | 'dropdown' | 'textarea' | 'label' | 'region'
+    'string' | 'number' | 'checkbox' | 'dropdown' | 'textarea' | 'label' | 'region' | 'environment'
 
 
 export type DynamicKVInput = {
   key: string;
-  value: string | number | boolean | string[] | null;
+  value: string | number | boolean | string[] | any | null;
```

Comment:
thats correct, i will replace with `EnvironmentDefinition<any>`

Reply:
I tried doing this and it resulted in a circular dependency error, since the library that component is in depends on this library

Reply:
Moving `ui-selectors` out of the base blueprint into its own library might make some sense.  This would allow us to support wizard-aware types defined in component SDKs and drive alignment between the components and the wizard options.

Reply:
We discussed, and its probably better to move any supported definitions to the base blueprint, and then have the components just reference those definitions.

Reply:
Great, agreed, should I include those changes in this PR

Reply:
No, lets not block on that